_docs/examples/ios.md

```diff
@@ -112,13 +114,15 @@ Memory.readUtf8String(data.bytes(), data.length());
 2. Converting NSData to Binary Data
 -
 ```
+var data = new ObjC.Object(args[2]);
 Memory.readByteArray(data.bytes(), data.length());
 ```
 
 3. Iterating an NSArray
 -
 ```
-var count = array.count();
+var array = new ObjC.Object(args[2]);
+var count = array.count().valueOf();  // Be sure to use valueOf!
```

Comment:
Maybe we should expand on that. E.g.: `Be sure to use valueOf() as NSUInteger is a Number in 32-bit processes, and UInt64 in 64-bit processes. This coerces it into a Number in the latter case.`